libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -207,12 +211,26 @@
 #define HAL_BOARD_TERRAIN_DIRECTORY "/var/APM/terrain"
 #define HAL_INS_DEFAULT HAL_INS_MPU9250
 #define HAL_BARO_DEFAULT HAL_BARO_MS5611_SPI
-#define HAL_COMPASS_DEFAULT HAL_COMPASS_AK8963_MPU9250  
+#define HAL_COMPASS_DEFAULT HAL_COMPASS_AK8963_MPU9250
+#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_BEBOP
+#define HAL_BOARD_LOG_DIRECTORY "/data/ftp/internal_000/APM/logs"
+#define HAL_BOARD_TERRAIN_DIRECTORY "/data/ftp/internal_000/APM/terrain"
```

Comment:
Last week we added support for runtime configuration of these directories. Right now it's only based on command line parameters, but in future it will come from a configuration file. I'm wondering if this part could already be dropped and you then start the binary by passing the right options.